guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -2,14 +2,15 @@
 
 = Configuring Repositories
 
-Use this procedure to enable the repositories that are required to install {ProductName}. Choose from the available list which operating system and version you are installing on:
+Use this procedure to enable the repositories that are required to install {ProductName}.
+Choose from the available list which operating system and version you are installing on:
 
 ifdef::foreman-el,katello[]
 * xref:#repositories-centos-8[CentOS 8]
 * xref:#repositories-centos-7[CentOS 7]
 endif::[]
 ifdef::foreman-el,katello,satellite[]
-* xref:#repositories-rhel-7[{RHEL} 7]
+* xref:#repositories-rhel-7[RHEL 7]
```

Comment:
You are still making a change here that I don't see necessary. `{RHEL}` is an attribute expaning to "Red Hat Enterprise Linux": https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L90

Reply:
Sorry about that...that one was just me not thinking.